src/vs/workbench/browser/labels.ts

```diff
@@ -492,31 +492,33 @@ class ResourceLabelWidget extends IconLabel {
 		if (!options.forceLabel && !isSideBySideEditor && resource?.scheme === Schemas.vscodeNotebookCellOutput) {
 			const notebookDocument = this.notebookDocumentService.getNotebook(resource);
 			const outputUriData = extractCellOutputDetails(resource);
-			if (!outputUriData?.notebook || !outputUriData.cellFragment) {
-				return;
-			}
-			const cellUri = outputUriData.notebook.with({
-				scheme: Schemas.vscodeNotebookCell,
-				fragment: outputUriData.cellFragment
-			});
-			const cellIndex = notebookDocument?.getCellIndex(cellUri);
-			const outputIndex = outputUriData.outputIndex;
-
-			if (cellIndex !== undefined && outputIndex !== undefined && typeof label.name === 'string') {
-				label.name = localize(
-					'notebookCellOutputLabel',
-					"{0} • Cell {1} • Output {2}",
-					label.name,
-					`${cellIndex + 1}`,
-					`${outputIndex + 1}`
-				);
-			} else if (cellIndex !== undefined && typeof label.name === 'string') {
-				label.name = localize(
-					'notebookCellOutputLabelSimple',
-					"{0} • Cell {1} • Output",
-					label.name,
-					`${cellIndex + 1}`
-				);
+			if (outputUriData?.cellFragment) {
```

Comment:
in this file the only changes are 

 adding `if (outputUriData?.cellFragment) {`
changing `if (!outputUriData?.notebook || !outputUriData.cellFragment) {` -> `if (!outputUriData?.notebook) {`

everything else is just indented so that is what the diff is showing